
def create_error(received_data):
    DATA = int(received_data)
    print(DATA)
    